rfcs/0000_template.md

```diff
@@ -29,7 +50,18 @@ implementation to implement. This should get into specifics and corner-cases,
 and include examples of how the feature is used. Any new terminology should be
 defined here.
 
-# Drawbacks
+Below are a couple of suggested sections that you can tweak as you desire.
+
+## Terminology
+
+A glossary of new terms can be very helpful.
+
+## Architectural diagrams.
```

Comment:
nit: +1000 for architectural diagrams. However, this seems to encourage a dedicated section for architectural diagrams, and normally I've seen them included within the detailed design itself.

Reply:
Yea, that makes total sense (I proposed this section initially). I'd be on board with removing it and just maybe adding a sentence under detailed design that says something about encouraging architectural diagrams.

Reply:
FWIW, it currently also is under "Detailed design", the detailed design has the first-level heading `#` and this section has the second-level heading `##` under "Detailed design".

I moved it as a plain sentence under "Detailed design", as suggested.